frontend/src/metabase/reference/metrics/MetricQuestions.jsx

```diff
@@ -49,75 +43,71 @@ const mapStateToProps = (state, props) => ({
   metric: getMetric(state, props),
   table: getTable(state, props),
   metadata: getMetadata(state),
-  entities: getMetricQuestions(state, props),
-  loading: getLoading(state, props),
-  loadingError: getError(state, props),
 });
 
 const mapDispatchToProps = {
   ...metadataActions,
 };
 
-class MetricQuestions extends Component {
-  static propTypes = {
-    style: PropTypes.object.isRequired,
-    entities: PropTypes.object.isRequired,
-    loading: PropTypes.bool,
-    loadingError: PropTypes.object,
-    metric: PropTypes.object,
-    table: PropTypes.object,
-    metadata: PropTypes.object.isRequired,
-  };
+const getDescription = question => {
+  const timestamp = moment(question.getCreatedAt()).fromNow();
+  const author = question.getCreator().common_name;
+  return t`Created ${timestamp} by ${author}`;
+};
 
-  render() {
-    const { entities, style, loadingError, loading, table, metric, metadata } =
-      this.props;
+export const MetricQuestions = ({ style, table, metric, metadata }) => {
+  const {
+    data = [],
+    isLoading,
+    error,
+  } = useQuestionListQuery({
+    query: { f: "using_metric", model_id: metric.id },
```

Comment:
@ranquild and @kamilmielnik `metric` wasn't marked as required in this file (unlike `segment` in SegmentQuestions). I can add a guard `metric?.id` or I can mark it as required.

There is no way to reach this component by actually clicking in the UI. You first need a metric in order to see questions that use it.

BUT

There is a scenario that the old code didn't handle. What happens if someone manually enters `/reference/metrics/:random-non-existent-id/questions`? The answer (from `master` is this)
![image](https://github.com/metabase/metabase/assets/31325167/b0c646bc-ba59-4dfb-92c7-fdd241a11e6b)

Given that someone needs to intentionally enter the wrong id, and given that we didn't handle this even before these new changes, do you still want me to handle that case or continue ignoring it?

If "ignore it" is the answer - I'll just mark `metric` as required.
If "handle it" is the answer - which component or which design should I use for the empty, not found scenario?

Reply:
I would ignore this case in this PR

Reply:
Done https://github.com/metabase/metabase/pull/38481/commits/5e1ee05927b75baa4730321baee3f585e20ca2fa

Thank you